doc/backend_vars.asciidoc

```diff
@@ -176,7 +176,7 @@ VIRTIO_CONSOLE;boolean;1;Enable/disable virtio console. (@see `-device virtconso
 VIRTIO_CONSOLE_NUM;integer;1;Number of virtio consoles.
 QEMU_BALLOON_TARGET;integer;undef;The target guest RAM usage before a snapshot is taken. It is intended to speed up snapshots by forcing the guest to drop various caches. Setting this enables the virtio-balloon device which requires a kernel with a virtio-balloon driver. Setting this far below the RAM required by the guest will probably cause the guest to panic or deadlock. However it should be able to cope with it being set slightly below what is needed.
 QEMU_BALLOON_TIMEOUT;integer;5;Timeout for qemu balloon operations
-RAIDLEVEL;;;Set the raid level
+RAIDLEVEL;;;Set the raid level, affects NUMDISKS, will set 4 by default.
```

Comment:
instead of repeating the text can we just point to NUMDISKS instead?

Reply:
fixed